src/pages/Landing/Landing.jsx

```diff
@@ -0,0 +1,73 @@
+import { useNavigate } from "react-router-dom";
+import Button from "components/Button";
+import logo from "assets/logo.png";
+import landing1 from "assets/landing1.png";
+import home1 from "assets/Home-1.png";
+import home2 from "assets/Home-2.png";
+import home3 from "assets/Home-3.png";
+import "./Landing.scss";
```

Comment:
지금은 스타일을 직접 import 하는 방식으로 사용하셨어요!
이 경우 작성하신 css 파일이 전체 페이지의 head 태그로 들어가
다른 파일의 classname이 중복될 수 있습니다.

저희 팀 rule이 `module + scss` 사용이니 나중에 적용해보시면 좋을 것 같습니다.
사용은 이렇게 하실 수 있습니다!
```suggestion
import styles from "./Landing.scss";
```

클래스 네임을 `{styles.기존-클래스-네임}` 형식으로 사용하시면 됩니다.
```js
<div className={styles.CenterLine}></div>
```